test/bpf/check-complexity.sh

```diff
@@ -26,27 +26,28 @@ function annotate_section_names {
 	    -e "s/\(section '2\/12'\)/\1 (tail_call IPV6_TO_LXC_POLICY_ONLY)/" \
 	    -e "s/\(section '2\/13'\)/\1 (tail_call IPV4_TO_ENDPOINT)/" \
 	    -e "s/\(section '2\/14'\)/\1 (tail_call IPV6_TO_ENDPOINT)/" \
-	    -e "s/\(section '2\/15'\)/\1 (tail_call IPV4_NODEPORT_NAT)/" \
-	    -e "s/\(section '2\/16'\)/\1 (tail_call IPV6_NODEPORT_NAT)/" \
-	    -e "s/\(section '2\/17'\)/\1 (tail_call IPV4_NODEPORT_REVNAT)/" \
-	    -e "s/\(section '2\/18'\)/\1 (tail_call IPV6_NODEPORT_REVNAT)/" \
-	    -e "s/\(section '2\/19'\)/\1 (tail_call IPV4_ENCAP_NODEPORT_NAT)/" \
-	    -e "s/\(section '2\/20'\)/\1 (tail_call IPV4_NODEPORT_DSR)/" \
-	    -e "s/\(section '2\/21'\)/\1 (tail_call IPV6_NODEPORT_DSR)/" \
-	    -e "s/\(section '2\/22'\)/\1 (tail_call IPV4_FROM_HOST)/" \
-	    -e "s/\(section '2\/23'\)/\1 (tail_call IPV6_FROM_HOST)/" \
-	    -e "s/\(section '2\/24'\)/\1 (tail_call IPV6_ENCAP_NODEPORT_NAT)/" \
-	    -e "s/\(section '2\/25'\)/\1 (tail_call IPV4_FROM_LXC_CONT)/" \
-	    -e "s/\(section '2\/26'\)/\1 (tail_call IPV6_FROM_LXC_CONT)/" \
-	    -e "s/\(section '2\/27'\)/\1 (tail_call IPV4_CT_INGRESS)/" \
-	    -e "s/\(section '2\/28'\)/\1 (tail_call IPV4_CT_INGRESS_POLICY_ONLY)/" \
-	    -e "s/\(section '2\/29'\)/\1 (tail_call IPV4_CT_EGRESS)/" \
-	    -e "s/\(section '2\/30'\)/\1 (tail_call IPV6_CT_INGRESS)/" \
-	    -e "s/\(section '2\/31'\)/\1 (tail_call IPV6_CT_INGRESS_POLICY_ONLY)/" \
-	    -e "s/\(section '2\/32'\)/\1 (tail_call IPV6_CT_EGRESS)/"
+	    -e "s/\(section '2\/15'\)/\1 (tail_call IPV4_NODEPORT_NAT_EGRESS)/" \
```

Comment:
I've tried running (from the dev VM) the `verifier-test.sh`, and it's failing (the `master` branch works though):

```
sudo ./test/bpf/verifier-test.sh -v
...
libbpf: prog 'from_network': unrecognized ELF section name 'from-network'
libbpf: couldn't reuse pinned map at '/sys/fs/bpf/tc//globals/test_cilium_calls_65535': parameter mismatch
libbpf: map 'test_cilium_calls_65535': error reusing pinned map
libbpf: map 'test_cilium_calls_65535': failed to create: Invalid argument(-22)
libbpf: failed to load object './test/bpf/../../bpf/bpf_network.o'
Unable to load program

Reply:
Yes I'm actually also not able to make it passes locally. But looks like it passed with CI for `/test-1.16-4.9` Would you make a try for `k8s-1.22-kernel-5.4 (test-1.22-5.4)` ?

I have addressed all your comments but prefer to wait your back on this before force-pushing

Reply:
Not so bad, no? A question, how have you been able to get those information related to insns?

Reply:
Actually, `processed X insns` is the verifier complexity. Yep, it's already a nice improvement! I expect both to become even smaller after the split of `snat_v4_process()`.

Reply:
I've run the following:
```
docker run --name foobar --rm -ti --privileged -v $PWD:/cilium quay.io/cilium/cilium-ci:latest /bin/bash
mount -t bpf bpf /sys/fs/bpf
cd /cilium
make -C bpf/ KERNEL=netnext
./test/bpf/check-complexity.sh
```